app/javascript/components/widgets/traffic/panel.jsx

```diff
@@ -84,18 +91,19 @@ const Traffic = () => (
         return null;
       }
 
-      const [firstColumn, secondColumn] = splitAt(3, trafficCams);
+      const columnSize = Math.floor(trafficCams.length / 2);
+      const [firstColumn, secondColumn] = splitAt(columnSize, trafficCams);
 
       return (
         <>
           <Row>
             <Column>
-              {take(3, firstColumn).map(({ id, ...cam }) => (
+              {take(columnSize, firstColumn).map(({ id, ...cam }) => (
```

Comment:
does take make sense if we're using the whole list?
can this just be `firstColumn.map(...`?

Reply:
Oh I see, we want both columns to be the same size